expression/distsql_builtin.go

```diff
@@ -659,7 +659,7 @@ func convertUint(val []byte) (*Constant, error) {
 		return nil, errors.Errorf("invalid uint % x", val)
 	}
 	d.SetUint64(u)
-	return &Constant{Value: d, RetType: types.NewFieldType(mysql.TypeLonglong)}, nil
+	return &Constant{Value: d, RetType: &types.FieldType{Tp: mysql.TypeLonglong, Flag: mysql.UnsignedFlag}}, nil
```

Comment:
Do we need to add the flag information to other `convert` functions?

Reply:
@zimulala convertXXX may need to be written since expr.FieldType is not used,
but it would be better to do this in another PR.